### PR TITLE
Article Kestra Weaviate

### PR DESCRIPTION
**Pull request by darioradecic**
As usual, a thumbnail is needed for the article.

I've discussed the topic with Anna, there are some issues with deleting Weaviate records from Kestra that will be updated in future releases. Once resolved, I'll update the article to showcase a deletion example.

Let me know if you need anything else.

### CODE REVIEW

File: content/blogs/2023-10-31-kestra-weaviate.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+![Image 4 - Flow execution](/blogs/2023-10-31-kestra-weaviate/4.png)
+
+If you've entered Weavite credentials correctly, you'll see a success message. There's no output showing you the newly created schema, but we can turn to WCS for that.
+
+For example, you can run the following query to get all data from the `Movie` schema:
+
+![Image 5 - Weaviate console query](/blogs/2023-10-31-kestra-weaviate/5.png)
+
+There are no records available to show, but the fact that you didn't get an error means the schema was created successfully!
```
Comment (anna-geller) [inline content/blogs/2023-10-31-kestra-weaviate.md:123]:
```suggestion
The output validates that the schema has been created successfully. Next, let's insert some data.
```

File: content/blogs/2023-10-31-kestra-weaviate.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+The results of the query are available in the *Outputs* tab. In there, you can either preview or download the results:
+
+![Image 11 - Flow outputs](/blogs/2023-10-31-kestra-weaviate/11.png)
+
+Here's what you'll see after clicking on the *Preview* button:
+
+![Image 12 - Output preview](/blogs/2023-10-31-kestra-weaviate/12.png)
+
+And that's how easy it is to query a Weavite vector database from Kestra. Let's make a brief recap next.
```
Comment (anna-geller) [inline content/blogs/2023-10-31-kestra-weaviate.md:230]:
```suggestion
And that's how easy it is to query a Weaviate vector database from Kestra. Next, let's do a brief recap.
```

File: content/blogs/2023-10-31-kestra-weaviate.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+---
+
+## Summing up Kestra and Weaviate Integration
+Vector databases are quickly becoming a must-know technology for a wide range of IT professionals - from software developers to data engineers and data scientists. Most modern applications have some AI features built in, which means a vector database is the most adequate place to store things such as embeddings for machine learning models.
+
+Weaviate is one example of a well-implemented vector database, so we at Kestra wanted to give you a **familiar-looking interface** for communicating with it. 
+
+This article showed you how to do the most common operations on simple data objects, but you can do so much more for actual machine learning use cases. We recommend you go over their [example use cases and demos](https://weaviate.io/developers/weaviate/more-resources/example-use-cases) list to see what's possible, and think about how you could integrate their examples in your Kestra flows.
```
Comment (anna-geller) [inline content/blogs/2023-10-31-kestra-weaviate.md:239]:
```suggestion
This article showed you how to do the most common operations on simple data objects, but you can do so much more for actual machine learning use cases. We recommend you go over their [example use cases and demos](https://weaviate.io/developers/weaviate/more-resources/example-use-cases) list to see what's possible and build end-to-end LLM-powered workflows using Kestra.
```

File: content/blogs/2023-10-31-kestra-weaviate.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+Make sure to **stay tuned** to the [Kestra blog](https://kestra.io/blogs) for more up-to-date guides on orchestration, automation tools, trends in data management, and new features and plugins in our platform.
```
Comment (anna-geller) [inline content/blogs/2023-10-31-kestra-weaviate.md:241]:
```suggestion
Subscribe to our newsletter below for more guides on modern orchestration, automation, and trends in data management, as well as new features in our platform.
```

**Review by anna-geller — APPROVED** [review]
Great work! Some small suggestions only

**Review by MartinRst — APPROVED** [review]

**Pull request merged**